src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -134,7 +134,7 @@ def _estimate_local_error(
         local_pred_rv = Normal(pred_rv.mean, calibrated_proc_noise_cov)
         local_meas_rv, _ = self.gfilt.measure(local_pred_rv, t_new)
         error = local_meas_rv.cov.diagonal()
-        return np.sqrt(error)
+        return np.sqrt(np.abs(error))
```

Comment:
In principle, yes. It lead to fairly flaky tests in my case though, so I decided to accept this absolute value.

By the time square-root filtering is around (#316 ), this will be stabilised/undone anyway.

Reply:
Sure, if it's removed soon anyways then I don't mind having it in this PR :+1: